components/user_settings/display/user_settings_theme/custom_theme_chooser.jsx

```diff
@@ -214,30 +216,34 @@ export default class CustomThemeChooser extends React.PureComponent {
     toggleSidebarStyles = (e) => {
         e.preventDefault();
 
-        $(this.refs.sidebarStylesHeader).toggleClass('open'); // eslint-disable-line jquery/no-class
+        this.refs.sidebarStylesHeader.classList.toggle('open');
         this.toggleSection(this.refs.sidebarStyles);
     }
 
     toggleCenterChannelStyles = (e) => {
         e.preventDefault();
 
-        $(this.refs.centerChannelStylesHeader).toggleClass('open'); // eslint-disable-line jquery/no-class
+        this.refs.centerChannelStylesHeader.classList.toggle('open');
         this.toggleSection(this.refs.centerChannelStyles);
     }
 
     toggleLinkAndButtonStyles = (e) => {
         e.preventDefault();
 
-        $(this.refs.linkAndButtonStylesHeader).toggleClass('open'); // eslint-disable-line jquery/no-class
+        this.refs.linkAndButtonStylesHeader.classList.toggle('open');
         this.toggleSection(this.refs.linkAndButtonStyles);
     }
 
     toggleSection(node) {
-        if (UserAgent.isIos()) {
-            // iOS doesn't support jQuery animations
-            $(node).toggleClass('open'); // eslint-disable-line jquery/no-class
+        node.classList.toggle('open');
+
+        // set overflow after animation, so the colorchooser is fully shown
+        if (node.classList.contains('open')) {
```

Comment:
Could the overflow-y always be hidden? I'm not sure if setting it to visible after the animation changes anything since an opened section should be big enough that nothing overflows.

Reply:
I tried it without.
But if you open the color picker for lowest select, the color picker is cut off due its absolute placement.

Reply:
Ah, I missed that when fiddling with it locally. Good catch!